pcsx2-qt/Settings/GraphicsSettingsWidget.ui

```diff
@@ -1172,9 +1172,9 @@
           </widget>
          </item>
          <item row="1" column="1">
-          <widget class="QCheckBox" name="wildHack">
+          <widget class="QCheckBox" name="forceEvenSpritePosition">
            <property name="text">
-            <string extracomment="Wild Arms: name of a game series. Leave as-is or use an official translation.">Wild Arms Hack</string>
+            <string extracomment="Wild Arms: name of a game series. Leave as-is or use an official translation.">Force Even Sprite Position</string>
```

Comment:
You can probably remove the translator comment, since it no longer mentions wild arms

Reply:
Can do.